modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -517,7 +517,7 @@ hello-smart-connector
 +
 (2) `hello-smart-connector/smart-connector-it/src/test/munit/assertion-munit-test.xml`: An assertion operation that calls the XML SDK operation.
 +
-. Run `mvn clean install` in the `/hello-smart-connector` to create the plugin for the `Hello XML SDK` module.
+. Run `mvn clean install` in the `/hello-smart-connector` to create the plugin for the `Hello XML SDK` module. This will install a parent project as well as child projects.
 +
```

Comment:
Can we avoid future tense unless needed for clarity, and avoid unclear antecedents? "This command installs a parent project as well as..."

Reply:
I remember I made this pull request based on customer's confusion but It's kind of obvious that executing 'mvn clean install' will take care of parent as well. If you believe the wording is causing more confusion or is ambiguous then we can ignore it.

Reply:
If it caused customer confusion before (it's obvious to you but maybe not to all users), it may do so again. IMO, it's okay to just leave it as it is now.